src/routes/app/projects/git/[gitUrl]/+page.ts

```diff
@@ -0,0 +1,50 @@
+import type { GenericGitSource, GitProject } from '../../types';
+import type { PageLoad } from './$types';
+
+export const load: PageLoad = ({ params }) => {
+  const { gitUrl } = params;
+
+  const decodedGitUrl = new URL(decodeURI(gitUrl));
```

Comment:
Pasting the decoded URL on the browser would allow me to navigate to the project's page?

Reply:
Yeah 👍